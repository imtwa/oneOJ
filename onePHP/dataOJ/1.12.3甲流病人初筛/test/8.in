40
SINLS 38.3225 0
UYCFJ 37.7233 0
IOXQP 36.0797 1
WDNZW 37.9586 0
CCPXD 38.2733 1
EWNXQ 36.1609 1
KGGEM 37.1596 0
HCPVW 36.0409 0
ROMZV 37.8212 1
TMSMS 38.1932 1
WOVUU 39.5191 0
EVANT 36.8984 0
XFBAC 38.0644 0
AUNSK 36.6549 0
ZYEYU 39.4258 1
JQZIV 37.4412 0
CFBMF 37.6995 0
TZDHC 38.3017 0
CAAFA 37.7409 1
XEDRP 39.1518 0
CVOMX 36.9208 1
FFFOA 38.4277 0
LZYKU 38.1704 0
UZRUL 38.409 1
EBXLA 37.2997 0
YZSJI 36.0107 0
QFVCN 38.3943 0
ZTRXF 39.2307 1
LWKIG 36.6585 0
CEUVW 37.4858 1
YAVZA 37.1589 0
NQACF 38.032 1
PCDJP 39.6152 1
NHBUW 36.6038 0
HKNML 37.8612 0
DEZYD 37.393 1
UOJWC 37.6273 0
RTRSU 37.2243 1
FLFOR 37.3608 1
QFGHH 36.1691 0
