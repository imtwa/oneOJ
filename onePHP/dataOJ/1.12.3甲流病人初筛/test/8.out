CCPXD
ROMZV
TMSMS
ZYEYU
CAAFA
UZRUL
ZTRXF
NQACF
PCDJP
9